core/src/consensus/tendermint/engine.rs

```diff
@@ -472,6 +472,12 @@ fn give_additional_rewards<F: FnMut(&Address, u64) -> Result<(), Error>>(
     let sorted_validators = missed_signatures
         .into_iter()
         .map(|(address, (proposed, missed))| (address, Ratio::new(missed, proposed)))
+        // When one sees the Ratio crate's Order trait implementation, he/she can easily realize the
```

Comment:
// When one sees the Ratio crate's Order trait implementation, he/she can easily realize that the

Reply:
Accepted